<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>피글DAO 연동 자동매매 설정</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pigicon.ico</normaloff>pigicon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="bts_oneStock">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>매수 설정</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>64</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>회당 매수량</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="et_buyVolume">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>자동매매 적용</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbt_apply">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>적용</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rbt_group_apply</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rbt_noApply">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>미적용</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rbt_group_apply</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbt_group_apply"/>
 </buttongroups>
</ui>
